packages/ses/test/test-import-cjs.js

```diff
@@ -3,15 +3,25 @@ import { resolveNode } from './node.js';
 import '../ses.js';
 import { freeze, keys } from '../src/commons.js';
 
-function heuristicRequires(moduleSource) {
+function heuristicAnalysis(moduleSource) {
   const dependsUpon = {};
+  const exports = {};
   moduleSource.replace(
     /(?:^|[^\w$_.])require\s*\(\s*["']([^"']*)["']\s*\)/g,
     (_, id) => {
       dependsUpon[id] = true;
     },
   );
-  return keys(dependsUpon);
+  moduleSource.replace(/(?:^|[^\w$_.])exports\.(\w[\w\d]*)\s*=/g, (_, name) => {
+    exports[name] = true;
+  });
+  moduleSource.replace(/(?:^|[^\w$_.])module\.exports\s*=/g, () => {
+    exports.default = true;
+  });
+  return {
+    imports: keys(dependsUpon),
+    exports: keys(exports),
```

Comment:
These heuristics, which are good enough for these tests, are what the CJS lexer will replace.